docs/source/design/task_class_structure.txt

```diff
@@ -0,0 +1,25 @@
+                                            +--------+
```

Comment:
I don't think you have to draw this - we have s as Sphinx plug-in that renders a class hierarchy diagram